packages/react-components/src/components/anomaly-chart/loading-icon.tsx

```diff
@@ -1,14 +1,36 @@
-import Button from '@cloudscape-design/components/button';
+import { Box, Popover, Spinner } from '@awsui/components-react';
+import {
+  colorTextBodyDefault,
+  fontFamilyBase,
+} from '@cloudscape-design/design-tokens';
 import React from 'react';
 
 export const LoadingIcon = ({ loading }: { loading?: boolean }) => {
   if (!loading) return null;
 
   return (
-    <div style={{ position: 'absolute', right: 8, bottom: 2, zIndex: 1 }}>
-      <Button loading variant='icon' loadingText='Loading'>
-        Loading
-      </Button>
+    <div
+      style={{
```

Comment:
Could this styling be applied via `className`?

Reply:
fixed!